.changeset/old-months-attack.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-azure-devops': minor
+---
+
+Updated to support cases where only Azure Pipelines are being used by adding an annotation that get builds by definition name
```

Comment:
I'm still not always sure when things are breaking. I think I may be wrong in this case as the current `isAzureDevOpsAvialable` function is still there. I've just added a new one to handle the new annotation. I've updated the README to make this more clear. Any suggestion as to how I can improve the changeset details? Should I add the same information from the README?

Reply:
Yeah you are right, this should be `patch`. But still, just saying the annotation name rather than "an annotation" still helps a lot i think so you at least know what to look for

Reply:
Ok, cleaned this up and improved the changeset.